Horion/Module/Modules/Jesus.h

```diff
@@ -5,11 +5,13 @@ class Jesus :
 {
 private:
 	bool wasInWater = false;
```

Comment:
You can remove this, as it is no longer used

Reply:
I'm not sure what you mean. The jesus module is used in this repository.